source/sessions/tests/dirty-session-errors.yml

```diff
@@ -231,9 +236,46 @@ tests:
               $numberLong: "1"
           command_name: insert
           database_name: *database_name
+      - command_started_event:
+          command:
+            find: *collection_name
+            filter: {_id: -1}
+          command_name: find
+          database_name: *database_name
 
     outcome:
       collection:
         data:
           - {_id: 1}
           - {_id: 2}
+
+  - description: Dirty implicit session is discarded (read)
+
+    # Enable the failpoint with times:2 so that this test can pass with or
+    # without retryable reads.
```

Comment:
Just to confirm, `times: 2` isn't a concern if retryable reads are not available because the test runner is instructed to clear any fail points after each test.

Reply:
Yep!